charts/vsphere-csi/v2.1.0/questions.yaml

```diff
@@ -84,3 +84,10 @@ questions:
         label: Data Store URL
         description: URL of the data store to use for new volumes (If unspecified, any data store that matches the request will be selected).
         type: string
+
+  - variable: csiNode.prefixPath
+    label: Prefix Path for `/var/lib/kubelet`
+    description: For some operating systems including RancherOS, RKE prefixes `/var/lib/kubelet` with `/opt/rke`.  Add the prefix path of the location of /var/lib/kubelet
+    type: string
+    default: ""
+    group: Node
```

Comment:
I don't think this group name is right, but I also can't think of what it should be.

@cbron Any ideas? Basically, how to indicate where your kubernetes resources are stored on your nodes.

Reply:
Customization ? Or Node Configuration

Reply:
Seems like we chose `Node Configuration`, so resolving this comment